GUI/src/app/nav-bar/nav-bar.component.html

```diff
@@ -25,7 +25,8 @@
         </a>
         <div class="dropdown-menu" aria-labelledby="navbarDropdown">
           <a class="dropdown-item" href="#">PLACEHOLDER</a>
-          <a class="dropdown-item" href="#">PLACEHOLDER</a>
+          <a class="dropdown-item" routerLink="/upload">Upload report(s)</a>
```

Comment:
maybe change the text, using a variable instead of plain text. would also be easier for possible multi language support